rush.json

```diff
@@ -66,6 +66,11 @@
       "packageName": "@itwin/infrastructure-iot-react",
       "projectFolder": "packages/itwin/infrastructure-iot",
       "versionPolicyName": "infrastructure-iot"
+    },
+    {
+      "packageName": "@itwin/ec3-react",

```

Comment:
I think the project name should be `@itwin/ec3-widget-react`.

Reply:
project name has been changed